frontend/revive-gallery-app/src/components/home/Navbar.js

```diff
@@ -1,54 +1,79 @@
 import { useUser } from '../auth/UserContext';
-import '../../styles/home/Navbar.css'; 
+import '../../styles/home/Navbar.css';
 import categories from '../../data/categories';
 
 const Navbar = () => {
-    const { user, token } = useUser();
-    const [isDropdownOpen, setDropdownOpen] = useState(false);
-
-    const toggleDropdown = () => {
-        setDropdownOpen(!isDropdownOpen);
-    }
-  
-    return (
-      <nav className="navbar">
-        <div className="navbar-logo">
-          <Link to="/">
-            <img src={logoImage} alt="Revive Gallery Logo" />
-          </Link>
-        </div>
-        <ul className="navbar-links">
-          <li>
-            <Link to="/">Home</Link>
-          </li>
-          <li>
-            <Link to="/products">Products</Link>
-          </li>
-          <li className="dropdown">
-            <span onClick={toggleDropdown}>Categories</span>
-            {isDropdownOpen && (
-                <ul className="dropdown-content">
-                    {categories.map((category) => (
-                        <li key={category.name}>
-                            <Link to={`/products?categories=${category.name}`}>{category.name}</Link>
-                        </li>
-                    ))}
-                </ul>
-            )}
-          </li>
-          {user && token ? (
+  const { user, token, logout } = useUser();
+  const [isDropdownOpen, setDropdownOpen] = useState(false);
+
+  const toggleDropdown = () => {
+    setDropdownOpen(!isDropdownOpen);
+  };
+
+  return (
+    <nav className="navbar">
+      <div className="navbar-logo">
+        <Link to="/">
+          <img src={logoImage} alt="Revive Gallery Logo" />
+        </Link>
+      </div>
+      <ul className="navbar-links">
+        <li>
+          <Link to="/">Home</Link>
+        </li>
+
+        <li>
+          <Link to="/products">Products</Link>
+        </li>
+
+        <li className="dropdown">
+          <span onClick={toggleDropdown}>Categories</span>
+          {isDropdownOpen && (
+            <ul className="dropdown-content">
+              {categories.map((category) => (
+                <li key={category.name}>
+                  <Link to={`/products?categories=${category.name}`}>{category.name}</Link>
+                </li>
+              ))}
+            </ul>
+          )}
+        </li>
+
+        {user && token ? (
             <li>
               <Link to="/addproduct">Add Product</Link>
             </li>
           ) : null}
-          
-        </ul>
-      </nav>
-    );
-  };
-  
-  export default Navbar;
-  
\ No newline at end of file
+
+        {user && token ? (
+          <li className="dropdown">
+            <span onClick={toggleDropdown}>User Profile</span>
+            {isDropdownOpen && (
+              <ul className="dropdown-content">
+                <li>
+                  <Link to="/user/trustworthiness">Trustworthiness Score</Link>
+                </li>
+                <li>
```

Comment:
@SaiKiran0407 I think adding "My Products" option here would be more nice.

Reply:
Yes, I will add My Products, Edit Profile, Logout, etc in the other user story which is related to designing user profile